Events

kavitaithomas opened the pull request:
- Added SpinCo event
- Created another button and added our linktree to top of events page
- made buttons at top of events page responsive
- Changed to consistent margins, headings, and paragraphs
- moved all img assets for events into events folder under public to be more organized

Corresponds to Issue #34 , #17

kalefed reviewed (changes requested):
For your images, to simplify things, try to to just pass the name, because you can set the image src as follows (without importing). For example pass the string: "/events/jmeet_mentors", and then src will have the value: 

 src="/events/jmeet_mentors"
 
 this should simplify the code a lot.

kavitaithomas commented:
Thanks for the feedback <3

Updates:
- Image paths have been simplified. I removed the imports and passed the path directly into the img prop.
- Added the Women at Microsoft Fireside Chat Event
- Made a design update to 'event-box' component. It now include the location and time in one line underneath the title of the event. I thought this was more clear, as opposed to including that info in the description which made it longer and unclear. Lmk what you think!
- Formatted Prettier extension in my user settings so the code looks cleaner now. Thank you for the instructions in the Qwic Dev Guide that hellped a lot!

kalefed reviewed (approved):
looks good :))
